quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/ClusterConfigDistTest.java

```diff
@@ -17,19 +17,22 @@
 
 package org.keycloak.it.cli.dist;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
+import java.nio.file.Path;
+import java.util.function.Consumer;
 import org.junit.jupiter.api.Test;
-import org.keycloak.it.cli.StartCommandTest;
+import org.keycloak.it.junit5.extension.BeforeStartDistribution;
 import org.keycloak.it.junit5.extension.CLIResult;
 import org.keycloak.it.junit5.extension.DistributionTest;
+import org.keycloak.it.junit5.extension.RawDistOnly;
+import org.keycloak.it.utils.KeycloakDistribution;
 
 import io.quarkus.test.junit.main.Launch;
 import io.quarkus.test.junit.main.LaunchResult;
 
 @DistributionTest
+@RawDistOnly(reason = "Not possible to mount files using docker.")
```

Comment:
this would be possible in general with a few changes, btw, but don't know if it brings much benefit now to touch the dockerdist for that. :)

Reply:
Sure, you can mount files in docker. But not in our impl.

Reply:
Hi @pedroigor, @DGuhr 
did this comment mean it is impossible to mount ispn-config.xml on the docker run command?